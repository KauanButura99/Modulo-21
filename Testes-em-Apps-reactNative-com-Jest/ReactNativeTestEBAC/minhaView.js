import React from 'react'
import { Button, TextInput, View } from 'react-native'


export default function MinhaView() {
  return (
    <View>
      <TextInput
        value='Meu texto'
      />
      <Button
        title='clique aqui'
        onPress={cliqueAqui}
      />
    </View>
  )
}

export function cliqueAqui() {
  return true
}