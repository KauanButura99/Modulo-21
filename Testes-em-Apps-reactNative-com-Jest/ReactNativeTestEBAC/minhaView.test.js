import React from "react";
import rederer from 'react-test-renderer'
import MinhaView from "./minhaView";
import { cliqueAqui } from "./minhaView";
import { expect } from "@jest/globals";

test('verificar a renderização', () => {
  const tree = rederer.create(<MinhaView />).toJSON
  expect(tree).toMatchSnapshot()
})

test('vereficar o Botão cliqueAqui', () => {
  expect(cliqueAqui).toBeTruthy()
})