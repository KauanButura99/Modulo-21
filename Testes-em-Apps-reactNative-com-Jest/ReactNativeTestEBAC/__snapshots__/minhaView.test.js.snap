// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`verificar a renderização 1`] = `[Function]`;
